docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -304,10 +304,10 @@ contains the specified string:
 output.elasticsearch:
   hosts: ["http://localhost:9200"]
   indices:
-    - index: "warning-%{[beat.version]}-%{+yyyy.MM.dd}"
+    - index: "warning-%{[{beat_version_key}]}-%{+yyyy.MM.dd}"
```

Comment:
These two attributes do not substitute as-is. Line 302 needs to be changed to allow substitution:
```
// old
["source","yaml"]
// new
["source","yaml",subs="attributes"]
```